docs/installation/sources/sources_windows.rst

```diff
@@ -0,0 +1,241 @@
+
+After downloading these packages, open an administrative shell with *PowerShell* and execute the following command:
+
+.. code-block:: bash
+
+    choco install -y -s <PATH_TO_DOWNLOADS> asio tinyxml2
+
+where :code:`<PATH_TO_DOWNLOADS>` is the folder you downloaded the packages into.
+
+Python
+^^^^^^
+
+Download and install the Python_ version that better fits your requirements.
+
+CMake, pip, wget and git
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can download and install CMake_, pip_, wget_ and git_ by following the instructions on the respective
+websites.
+Once installed, you need to add the path to these executables to your :code:`PATH` from the
+*Edit the system environment variables* control panel.
+
+OpenSSL
+^^^^^^^
+
+You can download the latest OpenSSL version for Windows at this webpage_.
+Download and use the installer that fits your requirements.
+After installing, add the environment variable :code:`OPENSSL_ROOT_DIR` pointing to the installation root directory.
+
+For example:
+
+.. code-block:: bash
+
+   OPENSSL_ROOT_DIR=C:\Program Files\OpenSSL-Win64
+
+Gtest
+^^^^^
+
+.. note::
+
+    By default, *eProsima Fast DDS* does not compile tests.
+    You can activate them by adding the :code:`-DPERFORMANCE_TESTS=ON` flag when calling colcon_ or CMake_
+    (for details, see below).
+
+You can find information on how to install Gtest at this `link <https://github.com/google/googletest>`_.
+
+.. _colcon_installation_windows:
+
+Colcon Installation
+-------------------
+
+colcon_ is a command line tool to build sets of software packages.
+This section explains how to use it to compile easily *eProsima Fast DDS* and its dependencies.
+
+.. important::
+
+    You need to run colcon within a Visual Studio prompt. To do so, launch a *Developer Command Prompt* from the
+    search engine.
+
+You can then install the ROS2 development tools (colcon_ and vcstool_):
+
+.. code-block:: bash
+
+    pip install -U colcon-common-extensions vcstool
+
+and add the path to the :code:`vcs` executable to your :code:`PATH` from the
+*Edit the system environment variables* control panel.
+
+.. note::
+
+    If this fails due to an Environment Error, add the :code:`--user` flag to your installation.
+
+Now, create a colcon_ workspace and download the repos file that will be used to install *eProsima Fast DDS* and
+its dependencies:
+
+.. code-block:: bash
+
+    $ mkdir Fast-DDS-ws && cd Fast-DDS-ws
+    $ wget https://raw.githubusercontent.com/eProsima/Fast-RTPS/master/fastrtps.repos
+    $ mkdir src
+    $ vcs import src < fastrtps.repos
+
+Finally, use colcon_ to compile all software:
+
+.. code-block:: bash
+
+    colcon build
+
+To run an *eProsima Fast DDS* instance, you need to source the colcon overlay with the command:
+
+.. code-block:: bash
+
+    setup.bat
+
+.. important::
+
+    The sourcing of the local colcon overlay is required every time the colcon workspace is opened in a new shell
+    environment to run an *eProsima Fast DDS* instance.
```

Comment:
There is no instance of the library. However, there are instances of applications using the library.

Reply:
yes, now I got it.